app/models/email.rb

```diff
@@ -1,3 +1,11 @@
+# == Schema Information
```

Comment:
Not sure if we want this here, since Emails are not saved into the DB afaik?
Not sure if its bad either